### PR TITLE
Add missing ethernet properties to QML types

### PR DESCRIPTION
**Pull request by LaakkonenJussi**
These were missing from earlier changes for ethernet.

### CODE REVIEW

File: plugin/plugins.qmltypes [hunk 1]
```diff
@@ -111,6 +111,7 @@ Module {
         Property { name: "offlineMode"; type: "bool" }
         Property { name: "defaultRoute"; type: "NetworkService"; isReadonly: true; isPointer: true }
         Property { name: "connectedWifi"; type: "NetworkService"; isReadonly: true; isPointer: true }
+        Property { name: "connectedEthernet"; type: "NetworkService"; isReadonly: true; isPointer: true }
```
Comment (inzanity) [inline plugin/plugins.qmltypes]: qmlplugindump would put this after "connectingWifi", so I suggest manual edit also puts it there.

**Review by inzanity — COMMENTED** [review]
The "ethernetServicesChanged" signal is still missing from NetworkManager.

File: plugin/plugins.qmltypes (hunk 1 above)
Comment (LaakkonenJussi) [reply plugin/plugins.qmltypes]: Done. Never heard about that binary even and on SDK it apparently does not work properly..

**Comment by LaakkonenJussi** [comment]
> The "ethernetServicesChanged" signal is still missing from NetworkManager.

Missed this comment and that signal apparently. Added now.

**Review by inzanity — APPROVED** [review]
LGTM.

**Pull request merged**